dGame/dBehaviors/ProjectileAttackBehavior.cpp

```diff
@@ -31,14 +31,6 @@ void ProjectileAttackBehavior::Handle(BehaviorContext* context, RakNet::BitStrea
 		return;
 	}
 
-	if (m_useMouseposit) {
-		NiPoint3 targetPosition = NiPoint3::ZERO;
-		if (!bitStream->Read(targetPosition)) {
-			Game::logger->Log("ProjectileAttackBehavior", "Unable to read targetPosition from bitStream, aborting Handle! %i", bitStream->GetNumberOfUnreadBits());
-			return;
-		};
-	}
-
```

Comment:
Is this not part of the structure? Check that Cannon Cove still works.

Reply:
This is apart of the structure after further investigation.  It looks to be based off of projectile_type instead of m_useMouseposit instead however as alongwith the plunger gun, lot 4799 also failed on main, which has a projectile_type of 2 as opposed to Cannon Coves 1.

Tested that 4799 and the plunger gun and the shooting gallery all properly use their skills and do not fault.